back-end/lms/models/student.py

```diff
@@ -32,7 +32,56 @@ class Student(Person):
     birthdate = models.DateField()
     program = models.ForeignKey(Program, models.DO_NOTHING)
     status = models.ForeignKey(Status, models.DO_NOTHING)
-    photo = models.CharField(max_length=128, blank=True, null=True)
+    photo = models.ImageField(upload_to = '', max_length=128)
+    place_birth = models.TextField()
+    region = models.TextField()
+    adress = models.TextField()
+    pasport = models.TextField() #можно числовой, но проще так
+    pasport_ufms = models.TextField()
+    pasport_ufms_code = models.TextField()
+    pasport_date = models.DateField()
+    citizence = models.TextField()
+    fio_rod = models.TextField()
+    hse_id = models.TextField()
+    faculty = models.TextField()
+    edu_program = models.TextField()
+    hse_group = models.TextField()
+    #mother
+    mother_fio = models.TextField()
+    mother_birthday = models.DateField()
+    mother_place_of_birth = models.TextField()
+    mother_citizence = models.TextField()
+    
+    #father
+    father_fio = models.TextField()
+    father_birthday = models.DateField()
```

Comment:
There is a [`DateField`](https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.DateField) for this.

Reply:
...and he is using it. Isn't he?

Reply:
Right, missed trying to parse through all these `TextField`s.